packages/codegen-ui/lib/types/data.ts

```diff
@@ -76,3 +85,9 @@ export type GenericDataSchema = {
 
   nonModels: { [nonModelName: string]: GenericDataModel };
 };
+
+export type DataStoreModelInfo = {
```

Comment:
Maybe we do not want this because we're in effect converting DataStore to Generic, only to make a DataStore type out of it again

Reply:
Yea we'll need to create a follow up pr to use this new generic schema